docs/configuration/configuration.md

```diff
@@ -294,7 +294,7 @@ See below for the configuration options for Azure discovery:
 subscription_id: <string>
 # Optional tenant ID. Only required with authentication_method OAuth.
 [ tenant_id: <string> ]
-# Optional client ID. Only required with authentication_method OAuth.
+# Optional client ID. Required with authentication_method OAuth, optional for authentication_method ManagedIdentity.
```

Comment:
I would shorten it: `Optional client ID. Required with authentication_method OAuth.`

Reply:
All right.